Card/src/main/java/com/paypal/android/card/CardLifeCycleObserver.kt

```diff
@@ -0,0 +1,19 @@
+package com.paypal.android.card
+
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+
+internal class CardLifeCycleObserver(private val cardClient: CardClient) : DefaultLifecycleObserver {
+
+    private var isFirstEventHandled = false
+
+    override fun onResume(owner: LifecycleOwner) {
+        super.onResume(owner)
+        // We want to ignore any previous onResume events before this class was instantiated
```

Comment:
Was there an actual bug with multiple `onResume` events? Browser switch should only deliver once.

Reply:
Its an issue with `lifecycle` in general. Like when I add an `Observer` to a lifecycle, it will immediately emmit the latest event, which was `onResume`. So i was getting `onResume` twice.